main.go

```diff
@@ -32,11 +32,11 @@ func main() {
 	}
 
 	r := gin.Default()
-	r.GET("/quotes", getRandomQuoteSQL)
-	r.GET("/quotes/:id", getQuoteByIdSQL)
-	r.POST("/quotes", addQuoteSQL)
-	r.DELETE("/quotes/:id", deleteQuote)
-	r.Run("0.0.0.0:8080")
+	r.GET("/quote", getRandomQuoteSQL)
+	r.GET("/quote/:id", getQuoteByIdSQL)
+	r.POST("/quote", addQuoteSQL)
+	r.DELETE("/quote/:id", deleteQuote)
+	r.Run("0.0.0.0:8081")
```

Comment:
I had to change this to 8080 to run both your API servers
```suggestion
	r.Run("0.0.0.0:8080")
```